src/Core/AdminConsole/Models/Data/EventIntegrations/RepositoryListenerConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Bit.Core.Settings;
+
+namespace Bit.Core.AdminConsole.Models.Data.EventIntegrations;
+
+public class RepositoryListenerConfiguration(GlobalSettings globalSettings)
+    : ListenerConfiguration(globalSettings), EventListenerConfiguration
+{
+    public string EventQueueName
+    {
+        get => _globalSettings.EventLogging.RabbitMq.EventRepositoryQueueName;
+    }
+
+    public string EventSubscriotionName
```

Comment:
```suggestion
    public string EventSubscriptionName
```

Just check all of these.